### DISCUSSION TITLE
Register only the fields that are updated

### DISCUSSION THREAD

**Discussion by Kublick** [react-hook-form/react-hook-form#2431] [Q&A] [answered]
Hello

I want to check if there is an option to only submit the fields that are updated in a form ; I saw there is unregister but it seems you have to specify exactly which field has to be unregistered.

Lets say I retrieve the following values from backend:
- Name
- Lastname
- Phone
- Address

I prefill the form with reset ; then update the phone number and Address ; on this case when I hit submit It would only send the updated phone and address.

**Comment by bluebill1049** [comment, marked as answer]
you will have to do a diff at submit level.

**Reply by Kublick** [reply]
ok makes sense ;  it will be way too verbose in order to compare each fields in the forms I'm making :)
